param/__init__.py

```diff
@@ -150,6 +151,109 @@ def param_union(*parameterizeds, **kwargs):
     return d
 
 
+def keywords_to_params(**kwargs):
```

Comment:
guess_param_types? (to make it parallel to guess_bounds)?

Reply:
Addressed in d8dd921